src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala

```diff
@@ -30,6 +30,9 @@ case class BlockchainApiRoute(readersHolder: ActorRef, ergoSettings: ErgoSetting
 
   private val paging: Directive[(Int, Int)] = parameters("offset".as[Int] ? 0, "limit".as[Int] ? 5)
 
+  /**
+    * Total number of boxes/transactions that can be requested at once
+    */
   private val MaxItems = 16384
```

Comment:
ok, but why this exact number is maximum. What is you motivation for choosing it?

Reply:
MaxHeaders is set to this number in BlocksApiRoute to "avoid too heavy requests", so I thought it is reasonable to use as a limit

Reply:
Again, this explanation better be in ScalaDoc.